src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -584,89 +584,21 @@ public static (SyntaxToken openBrace, SyntaxToken closeBrace) GetBrackets(this S
 
         public static SyntaxTokenList GetModifiers(this SyntaxNode member)
```

Comment:
>GetModifiers [](start = 38, length = 12)

This method now covers more cases (GlobalStatementSyntax, NamespaceDeclarationSyntax, for example). Maybe be easy enough to cover in test to avoid regressions.
Same comment for `WithModifiers` below.

Reply:
I checked, and we have tests for this in SyntaxGeneratorTests.

Reply:
Also added tsts for attributes.